src/toast/scripts/benchmarking_utilities.py

```diff
@@ -804,7 +804,8 @@ def run_mapmaker(job_ops, args, tmpls, data):
     job_ops.mapmaker.output_dir = args.out_dir
 
     # Run the map making
-    job_ops.mapmaker.apply(data)
+    # NOTE: we set `use_accel=False` as templates cannot be initialized on device
+    job_ops.mapmaker.apply(data, use_accel=False)
```

Comment:
This should be `None` instead of `False`.  Setting it to False will disable GPU use for the entire mapmaking.  The specific case of building the RHS on the CPU is already handled separately in this PR.

Reply:
Fixed!